redfish-core/lib/processor.hpp

```diff
@@ -900,6 +954,275 @@ inline void
     }
 }
 
+template <typename Handler>
+inline void getProcessorPaths(const std::shared_ptr<bmcweb::AsyncResp>& aResp,
+                              const std::string& processorId, Handler&& handler)
+{
+    crow::connections::systemBus->async_method_call(
```

Comment:
Use common function dbus::utility::getSubTreePaths() for this async_method_call().